internal/server/server.go

```diff
@@ -10,18 +10,17 @@ import (
 	"net"
 	"net/http"
 	"os"
-	"os/exec"
 	"syscall"
 	"time"
 
 	"github.com/blakewilliams/remote-development-manager/internal/client"
-	"github.com/blakewilliams/remote-development-manager/internal/clipboard"
+	"github.com/blakewilliams/remote-development-manager/internal/hostservice"
 )
 
 type Server struct {
+	hostservice.Service
```

Comment:
```suggestion
	host: hostservice.Service
```

Similar to the above, thoughts on using an explicit field instead of embedding? I think this might also read nicely in code, since it's `server.host.Copy`, `server.host.Paste`, etc.

Reply:
Agreed, I like this.